### PR TITLE
RealMe 6 Overlay

### PR DESCRIPTION
**Pull request by LooKeR-learner**

### CODE REVIEW

File: Realme/6/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?><manifest xmlns:android="http://schemas.android.com/apk/res/android" package="me.phh.treble.overlay.realme.6" platformBuildVersionCode="27" platformBuildVersionName="8.1.0">
+    <overlay android:isStatic="true" android:priority="96" android:requiredSystemPropertyName="ro.build.overlay.deviceid" android:requiredSystemPropertyValue="+RMX2001" android:targetPackage="android"/>
```
Comment (phhusson) [inline Realme/6/AndroidManifest.xml:2]: split it on multi lines like the other overlays for a better readability

**Review by phhusson — COMMENTED** [review]
What does it fix? What have you tested?

File: Realme/6/AndroidManifest.xml (hunk 1 above)
Comment (LooKeR-learner) [reply Realme/6/AndroidManifest.xml:2]: okay sir

**Comment by LooKeR-learner** [comment]
> What does it fix? What have you tested?

this fix notch and statusbar height and maybe headphone jack issue

**Pull request merged**